grouper/models/group_edge.py

```diff
@@ -4,6 +4,7 @@
 from sqlalchemy.orm import relationship
 
 from grouper.expiration import add_expiration, cancel_expiration
+from grouper.fe.template_util import _make_date_obj, _utcnow
```

Comment:
If we're using these methods here, let's move them elsewhere. Importing front-end code in a model is backwards.